### DISCUSSION TITLE
Command to remux the extracted DV7.EL_RPU file back into the MKV

### DISCUSSION THREAD

**Discussion by cannikin** [nekno/DV7toDV8#11] [Q&A] [answered]
Hello! Thanks so much for this script, it's a life saver! :) I'm going to start remuxing my Profile 7 files to Profile 8 for Infuse, and am planning on keeping the resulting `DV7.EL_RPU.hevc` file around in case support for 7 is ever added in the future, and I can use that extra EL goodness.

What would the process of recreating the Profile 7 file look like? Do I have to first extract a DV8.BL_RPU.hevc file from the the Profile 8 MKV and then combine DV8.BL_RPU.hevc + DV7.EL_RPU.hevc before finally merging that resulting file back into MKV? What's the opposite of this command for merging them back together?

```
"$doviToolPath" demux --el-only "$BL_EL_RPU_HEVC" -e "$DV7_EL_RPU_HEVC"
```

Thanks for any help!

**Comment by nekno** [comment, marked as answer]
Yep, first extract the HEVC track from the MKV file (call it, e.g., `DV8.BL_RPU.hevc`). Since FFMPEG is unreliable in demuxing DV HEVC files, I'd recommend using `mkvextract`, like the script does.

Then remux the `DV8.BL_RPU.hevc` and `DV7.EL_RPU.hevc` files using `dovi_tool`'s **mux** command. You don't have to worry about the DV8 RPU present in the `DV8.BL_RPU.hevc` file, as `dovi_tool` will automatically overwrite the RPU with the DV7 RPU from the `DV7.EL_RPU.hevc` file when remuxing.

```
"$doviToolPath" mux --bl "DV8_BL_RPU.hevc" --el "DV7.EL_RPU.hevc" -o "DV7.BL_EL_RPU.hevc"
```

Then remux the `DV7.BL_EL_RPU.hevc` file with your other MKV tracks.

**Reply by cannikin** [reply]
Thanks so much! Here's hoping that one day Infuse supports the enhanced brightness and other goodies in that EL data!
